.github/workflows/destroy_aws_environment.yml

```
@@ -0,0 +1,193 @@
+        type: boolean
+        default: false
+
+permissions:
+  issues: write
+
+jobs:
+  destroy-plan:
+    runs-on: ubuntu-latest
+    environment: Development
+    steps:
+      - name: Get Environment Name for ${{ vars.ENV_NAME }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)